### PR TITLE
doc: eac config review

### PR DESCRIPTION
**Pull request by scovertb**

### CODE REVIEW

**Comment by scovertb** [comment]
CU-85zry1kcw

File: docs/easy_anticheat_configuration.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ The EOS plugin provides a method of automating EAC installation as part of the b
     1) If you want to customize the config for EAC integrity tool, you can copy the default config from `Packages/Epic Online Services Plugin for Unity/Editor/Standalone/anticheat_integritytool.cfg` to your desired location, customize it, and set `Path to EAC Integrity Tool Config` to the new file.
```
Comment (andrew-hirata-playeveryware) [inline docs/easy_anticheat_configuration.md]: On line 11: should it say "With the plugin installed"?

Comment (scovertb) [reply docs/easy_anticheat_configuration.md]: I changed that, but it's phrased that way in configuring_the_eos_plugin.md as well

Comment (Raxnam) [inline docs/easy_anticheat_configuration.md]: I feel it should read "the config for the EAC integrity tool"

**Review by Raxnam — CHANGES_REQUESTED** [review]
looks good, links to epic's documentation are a good idea

**Review by Raxnam — APPROVED** [review]
looks good

**Pull request merged**